backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
FWIW, personally I already use `.promise`, as that was what was stated in the upgrade guide. Didn't even know I could do `.then` on the view until this issue popped up.
On the other hand, I DO utilize the fact that `this` refers to the view`. I'll gladly refactor to using the argument, as I'm no real fan of magical `this`. :laughing:

Want me to make a PR for `reoslveWith` (if I can make it wor)?